pkg/artifactory/provider.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 // Version for some reason isn't getting updated by the linker
-var Version = "2.6.18"
+var Version = "2.6.20"
```

Comment:
Mmm, because of the changes to make, you can set this to anything you like and the linker should overwrite it. Meaning, no need to ever change it again